src/Pos/Wallet/Web/Api.hs

```diff
@@ -104,46 +136,47 @@ type WalletApi =
      :> "txs"
      :> "payments"
      :> Capture "passphrase" CPassPhrase
-     :> Capture "from" CAddress
-     :> Capture "to" CAddress
+     :> Capture "from" CWalletAddress
+     :> Capture "to" (CAddress Acc)
      :> Capture "amount" Coin
-     :> Post '[JSON] (Either WalletError CTx)
+     :> Post '[JSON] (Either WalletError [CTx])
     :<|>
     -- TODO: for now we only support one2one sending. We should extend this to support many2many
      "api"
      :> "txs"
      :> "payments"
      :> Capture "passphrase" CPassPhrase
-     :> Capture "from" CAddress
-     :> Capture "to" CAddress
+     :> Capture "from" CWalletAddress
+     :> Capture "to" (CAddress Acc)
      :> Capture "amount" Coin
      :> Capture "currency" CCurrency
      :> Capture "title" Text
      :> Capture "description" Text
-     :> Post '[JSON] (Either WalletError CTx)
+     :> Post '[JSON] (Either WalletError [CTx])
```

Comment:
@Martoon-00 Something is off here. What is account ? As I understand WalletSet is set of wallets/accounts, right? We can send money from one wallet/account to some address. Can you give me an example of using this endpoint ? Because I can't seem to think a way to create multiple transactions when user is expecting to have exactly one transaction created when he is doing action "Send from wallet X into address Y amount Z".